images/kindnetd/go-mod-k8s.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
This script comes from https://github.com/kubernetes/kubernetes/issues/79384#issuecomment-521493597

Reply:
We already have this working, you just set the version in requirements and it will be replaced with a pseudo version

Why are we changing the version anyhow?

Reply:
https://github.com/kubernetes-sigs/kind/pull/947/commits/c34ac87fb5d6928d4a5a20e93c7f8d8e8ae84cfc#r334287177

Reply:
using go-iptables we no longer need to use kubernetes as a module